### PR TITLE
feat:[CDS-92733]: Multiple runtime support for AWS Serverless and SAM

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/serverless-lambda-cd-quickstart.md [hunk 1]
```diff
@@ -225,7 +225,21 @@ Currently, Serverless functions can't be deployed with an OIDC-enabled AWS conne
 
 ### Containerized step images
 
-Currently, for the containerized Serverless steps Harness provides, the base images Harness provides on Docker Hub have Node version 12.20.0 and Serverless version 3.30.1 installed. These are Linux AMD64 images.
+Currently, Harness support multiple runtime images for **nodejs20**, **nodejs18** and **java17**. These are Linux AMD64 images.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/serverless-lambda-cd-quickstart.md:228]:
```suggestion
Currently, Harness support multiple runtime images for **nodejs20**, **nodejs18**, and **java17**. These are Linux AMD64 images. 
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/serverless-lambda-cd-quickstart.md [hunk 2]
```diff
@@ -225,7 +225,21 @@ Currently, Serverless functions can't be deployed with an OIDC-enabled AWS conne
...
+There are two flavours of images available first with serverless installed and other without serverless installed that you can use.
+
+ Runtimes | With Serverless Installed | Without Serverless Installed
+| --- | --- | --- |
+| nodejs 20 | harness/serverless-plugin:nodejs20.x-3.39.0-1.0.0-beta-linux-amd64 | harness/serverless-plugin:nodejs20.x-1.0.0-beta-linux-amd64 |
+| nodejs 18 | harness/serverless-plugin:nodejs18.x-3.39.0-1.0.0-beta-linux-amd64 | harness/serverless-plugin:nodejs18.x-1.0.0-beta-linux-amd64 |
+| java 17 | harness/serverless-plugin:java17-3.39.0-1.0.0-beta-linux-amd64 | harness/serverless-plugin:java17-1.0.0-beta-linux-amd64 |
+
+
+Now, let's understand the runtime image one with serverless installed and one without serverless installed.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/serverless-lambda-cd-quickstart.md:238]:
```suggestion
Now, let's understand one runtime image with serverless installed and one without serverless installed. 
```

**Review by SushrutHarness — DISMISSED** [review]
Left a few suggestions. but otherwise LGTM

File: docs/continuous-delivery/deploy-srv-diff-platforms/serverless/static/multiple-runtime-image-serverless.png
Comment (piyush-bhuwalka) [inline docs/continuous-delivery/deploy-srv-diff-platforms/serverless/static/multiple-runtime-image-serverless.png:1]: Can we add a similar image for sam tags in sam documentation?

Comment (krishi0408) [reply docs/continuous-delivery/deploy-srv-diff-platforms/serverless/static/multiple-runtime-image-serverless.png:1]: Yup sure adding.

**Review by SushrutHarness — DISMISSED** [review]
LGTM

**Review by piyush-bhuwalka — DISMISSED** [review]

**Pull request merged**